### PR TITLE
Add gargvk81 edison file

### LINKED ISSUE #41: Create a new file for gargvk81 Edison, NJ
- [x] Create a branch
- [x] Add Commits
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by gargvk81**
closes #41

### CODE REVIEW

File: gargvk81-edison.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Edison, NJ
+
+### Sports Recommendations
+- Ymca
+- Baseball field
+- Roosevelt park
+
```
Comment (gargvk81) [inline gargvk81-edison.md:9]: only indoors?

File: gargvk81-edison.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+### Places to go
+- Top golf
+- Bowling
```
Comment (rkevinr) [inline gargvk81-edison.md:12]: Sounds good

**Review by gargvk81 — COMMENTED** [review]
great

**Review by rkevinr — APPROVED** [review]
sounds good

**Pull request merged**